src/IECoreMaya/ToMayaPlugConverter.cpp

```diff
@@ -68,12 +71,100 @@ ToMayaPlugConverterPtr ToMayaPlugConverter::create( const IECore::ObjectPtr src
 	return new ToMayaPlugConverter( src );
 }
 
-bool ToMayaPlugConverter::convert( MPlug &plug ) const
+template <typename DataType, typename UnitType>
+MStatus ToMayaPlugConverter::setPlugValue(MPlug &plug, boost::intrusive_ptr<const IECore::TypedData<DataType> > data)
 {
-	ConstObjectPtr toConvert = srcParameter()->getValidatedValue();
+	MStatus s;
```

Comment:
If we're not going to validate the status, lets just return it rather than storing it temporarily.

Reply:
Same for all the specializations below